tests/grouptype/test_grouptype.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+
+from sentry.grouptype.grouptype import GroupType, get_group_types_by_category
+from sentry.testutils import TestCase
+from sentry.testutils.silo import region_silo_test
+from sentry.types.issues import GroupCategory
+
+
+@region_silo_test
+class GroupTypeTest(TestCase):
+    def test_get_types_by_category(self):
+        assert get_group_types_by_category(GroupCategory.PERFORMANCE.value) == [
+            1001,
+            1004,
+            1006,
+            1007,
+            1008,
+            1010,
+            1011,
+            1012,
+        ]
+        assert get_group_types_by_category(GroupCategory.PROFILE.value) == [2000]
+        assert get_group_types_by_category(GroupCategory.ERROR.value) == [1]
```

Comment:
This test is a bit brittle, every time someone adds a new type we're going to have to update it.

How about we mock the registry, and register some test dataclasses here so that it never changes?